recipe/meta.yaml

```diff
@@ -78,6 +76,7 @@ test:
         - sites.sbx
         - sites.shp
         - sites.shx
+        - cropped.cub
```

Comment:
@michaelaye I am using your sample file in the test to avoid regression. Is that OK with you?